frontend/src/pages/CompareV2.test.tsx

```diff
@@ -73,23 +74,42 @@ describe('CompareV2', () => {
     return context;
   }
 
-  function newMockExecution(id: number): Execution {
+  function newMockExecution(id: number, displayName?: string): Execution {
     const execution = new Execution();
     execution.setId(id);
+    if (displayName) {
+      const customPropertiesMap: Map<string, Value> = new Map();
+      const displayNameValue = new Value();
+      displayNameValue.setStringValue(displayName);
+      customPropertiesMap.set('display_name', displayNameValue);
+      jest.spyOn(execution, 'getCustomPropertiesMap').mockReturnValue(customPropertiesMap);
+    }
     return execution;
   }
 
-  function newMockEvent(id: number): Event {
+  function newMockEvent(id: number, displayName?: string): Event {
     const event = new Event();
     event.setArtifactId(id);
     event.setExecutionId(id);
     event.setType(Event.Type.OUTPUT);
+    if (displayName) {
+      const path = new Event.Path();
+      const step = new Event.Path.Step();
+      step.setKey(displayName);
+      path.addSteps(step);
+      event.setPath(path);
+    }
     return event;
   }
 
-  function newMockArtifact(id: number): Artifact {
+  function newMockArtifact(id: number, confusionMatrix?: boolean): Artifact {
```

Comment:
NIT: use name `isConfusionMatrix` to follow convention that variable is a boolean.

Reply:
Done! ✅  Commit here: https://github.com/kubeflow/pipelines/pull/7943/commits/eb9faf5a4e8ddf4d7b8772d256feb7b47013b489.